frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -1189,70 +1321,55 @@ export const CreateOperand: React.FC<CreateOperandProps> = ({
     [customResourceDefinition, operandModel],
   );
 
+  const defaultSample = React.useMemo<K8sResourceKind>(
+    () =>
+      JSON.parse(_.get(csvAnnotations, 'alm-examples', '[]')).find(
+        (s: K8sResourceKind) => referenceFor(s) === operandModelReference,
+      ),
+    [operandModelReference, csvAnnotations],
+  );
+
+  const [buffer, setBuffer] = React.useState<K8sResourceKind>();
+
+  const onToggleEditMethod = React.useCallback(
+    (newBuffer) => {
+      setBuffer(newBuffer);
+      setMethod((currentMethod) => (currentMethod === 'yaml' ? 'form' : 'yaml'));
+      console.log(method);
+    },
+    [defaultSample],
+  );
+
+  const editor = React.useMemo(() => {
```

Comment:
Had issues with switching between yaml and form components when it was triggered by the child. I think this relates back to problems we've had where Firehose can cause issues with re-renders, but I might be wrong.